.env/.env.prod

```diff
@@ -50,4 +50,6 @@ GENERATE_SOURCEMAP=false
 REACT_APP_RECAPTCHA_SITE_KEY=6LfVR-0ZAAAAADFcqNM1P1IafKwQwN0E_l-gxQ9q
 
 REACT_APP_BITSKI_CLIENT_ID=7a543ec2-b55f-45d6-a5d4-0448c5a23485
-REACT_APP_BITSKI_CALLBACK_URL=https://audius.co/bitski-callback.html
\ No newline at end of file
+REACT_APP_BITSKI_CALLBACK_URL=https://audius.co/bitski-callback.html
+
+REACT_APP_OPENSEA_API_URL=https://collectibles.audius.co/api/v1
```

Comment:
can you also add to `configureLocalEnv.js`
This script generates .env.dev based on local backend services

Reply:
will do